src/common/ipc.ts

```diff
@@ -17,7 +17,11 @@ export const ipc_getDesktopAppPath = makeEndpoint.main(
   <{ path: string; exists: boolean }>_,
 );
 
-export const ipc_deleteFolder = makeEndpoint.main("deleteFolder", <{ path: string }>_, <SuccessPayload>_);
+export const ipc_migrateDolphin = makeEndpoint.main(
+  "deleteFolder",
```

Comment:
This event name should probably be updated.

```suggestion
  "migrateDolphin",
```